toolbox/gpu-operator/operand-build/build_all.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash -e
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source ${THIS_DIR}/../../_common.sh
+
+operands_dir=${THIS_DIR}/operands
+
+for operand_json in $(ls ${operands_dir}); do
+  exec ansible-playbook -e "$(< ${operands_dir}/${operand_json})" playbooks/build-operand.yml &
+done
+
+wait
```

Comment:
I think you'll need a way to catch errors in `ansible-playbook`

When you run this:
```
set -e
sleep 10 &
sleep 15 &
false &

wait
echo done
```
the error on `false` (= a playbook failing) is swallowed and the script completes successfully